package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "ecommerce-app-legends-bn",
+  "version": "1.0.0",
+  "description": "## Introduction",
+  "main": "index.js",
+  "scripts": {
+    "build": "babel src/index.js -d dist",
+    "start": "npm run build && nodemon dist/index.js",
+    "dev": "npm run build && nodemon --exec babel-node src/index.js",
```

Comment:
you don't need to build the app in development just run you ap with nodemon and babel. So remove the first command in the chain (`npm run build`).